client/src/components/controls/EventChart.vue

```diff
@@ -209,11 +226,12 @@ export default {
       if (remainder > 10) {
         return;
       }
-      const top = offsetY - (offsetY % 15);
+      const top = offsetY - (offsetY % 15) + 3;
```

Comment:
I'd complain about magic numbers, but this file is pretty much only magic numbers.  We should take care of that at some point, but now is not that time.